auction9-frontend/src/app/services/auction.service.ts

```diff
@@ -11,37 +11,44 @@ export class AuctionService {
 
   /* getActiveAuctions - returns all auctions with active state
    * Method: GET
-   * Path: /auctions 
+   * Path: /auctions
    */
   getActiveAuctions() {
     return this.http.get(`${environment.baseUrl}/auctions`, { responseType: 'json' }).toPromise();
   }
 
   /* getAuctionById - returns auction with specific ID
    * Method: GET
-   * Path: /auctions/${auctionId} 
+   * Path: /auctions/${auctionId}
    */
   getAuctionById(auctionId) {
     return this.http.get(`${environment.baseUrl}/auctions/${auctionId}`, { responseType: 'json' }).toPromise();
   }
-    
+
   /* addAuction - creates new auction
    * Method: POST
-   * Path: /auctions 
+   * Path: /auctions
    */
   addAuction(auction) {
     return this.http.post(`${environment.baseUrl}/auctions`, auction);
   }
 
-  /* getMyAuctions - return all auctions for current user
+  /* getAuctionBids - returns auction bids (users' bids history)
    * Method: GET
-   * Path: /myauctions 
-   */
-   // currently userId hard coded for testing purpose
-   // after SSO is implemented, this will be updated
+   * Path: auctions/{id}/bids */
+  getAuctionBids(auctionID) {
+    return this.http.get(`${environment.baseUrl}/auctions/${auctionID}/bids`, { responseType: 'json' }).toPromise();
+  }
+
+  /* getMyAuctions - return all auctions for current user
+  * Method: GET
+  * Path: /myauctions
+  */
+  // currently userId hard coded for testing purpose
+  // after SSO is implemented, this will be updated
   getMyAuctions() {
```

Comment:
Name of this method should be getUserAuctions, same goes for lambda name in serverless file and path.

Reply:
Okay.